corehq/apps/app_manager/templates/app_manager/partials/settings/app_settings.html

```diff
@@ -169,6 +169,7 @@ <h6>Example:</h6>
     {% if is_linked_app %}
       <li><a href="#linked-app" data-toggle="tab">{% trans "Linked Application" %}</a></li>
       <li><a href="#languages" data-toggle="tab">{% trans "Languages" %}</a></li>
+      <li><a href="#multimedia-tab" data-toggle="tab">{% trans "Multimedia" %}</a></li>
```

Comment:
This should check `master_is_remote`.

Reply:
Oops, nevermind, this is right. I was missing that the Download section shows for all linked apps, even though the new Sync section is only for remote linked apps.